.circleci/config.yml

```diff
@@ -48,20 +48,17 @@ jobs:
     docker:
       - image: circleci/openjdk:latest-node
       - image: selenium/standalone-chrome
-    parallelism: 2
```

Comment:
this was causing job to run same tests 2 times, causing increased response times

Reply:
@arieznikov 
This is interesting and a great find.  Would this parallelism be useful in running the unit test?

Reply:
units are already divided on job level